system/Validation/FormatRules.php

```diff
@@ -346,4 +346,29 @@ public function valid_date(?string $str = null, ?string $format = null): bool
 
         return $date !== false && $errors !== false && $errors['warning_count'] === 0 && $errors['error_count'] === 0;
     }
+
+    /**
+     * Token JWT Validator
+     *
+     * @param string $str String
+     */
+    public function valid_jwt(string $str = ''): bool
+    {
+        $newString = explode('.', $str);
+
+        if (count($newString) === 3) {
+            foreach ($newString as $index => $value) {
+                if ($index < 2) {
```

Comment:
Verifying the signature is of course beyond the scope of this rule but I think we should at least check that it is non-empty.